content/docs/how-to-guides/sqlx.md

```diff
@@ -52,32 +52,20 @@ In-line Javascript can be used anywhere SQL is written in order to dynamically m
 
 ### Built-in functions
 
-Built in functions have special functionality should be executed within [in-line Javascript](#in-line-javascript).
+Built in functions have special functionality and can be executed either within [in-line Javascript](#in-line-javascript) or [javascript blocks](#javascript-blocks).
 
-#### `ref()`
+For all built in functions, see [`ITableContext` in the API reference](/reference#ITableContext). Some useful examples can be found here:
 
-`ref()` enables you to easily reference another dataset in your project without having to provide the full SQL dataset name. `ref()` also adds the referenced dataset to the set of dependencies for the query.
+- [Reference another dataset in your project with `ref()`](datasets/#referencing-other-datasets).
```

Comment:
IMO the earlier sub-headings called out these examples a bit more nicely, fwiw these functions, especially `ref()` are going to be 99% of all context method invocations, i think it would be a good idea to continue calling them out loudly